<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1254</width>
    <height>710</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="verticalLayout_main">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_objects">
        <property name="rightMargin">
         <number>5</number>
        </property>
        <item>
         <layout class="QGridLayout" name="gridLayout_objects">
          <item row="0" column="5">
           <widget class="QPushButton" name="button_objects_add">
            <property name="text">
             <string>Erstellen</string>
            </property>
            <property name="autoDefault">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="2" colspan="3">
           <layout class="QHBoxLayout" name="horizontalLayout_object_button">
            <item>
             <widget class="QLineEdit" name="line_edit_object_name">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="toolTip">
               <string>Name der Objektvorgabe</string>
              </property>
              <property name="frame">
               <bool>true</bool>
              </property>
              <property name="echoMode">
               <enum>QLineEdit::Normal</enum>
              </property>
              <property name="placeholderText">
               <string>Name</string>
              </property>
              <property name="clearButtonEnabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="line_edit_abbreviation">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;die Abkürzung ist relevant für die Bauwerksstruktur.&lt;br/&gt;Sie sollte zwischen 1-10 Zeichen Lang sein und darf nur einmal vorkommen.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="placeholderText">
               <string>Abkürzung</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="lineEdit_ident_pSet">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Name des PropertySets in dem das Identifier Attribut sich befindet</string>
            </property>
            <property name="frame">
             <bool>true</bool>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Normal</enum>
            </property>
            <property name="placeholderText">
             <string>PropertySet</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_object_name">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>0</height>
             </size>
            </property>
            <property name="lineWidth">
             <number>1</number>
            </property>
            <property name="text">
             <string>Objektvorgabe</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLineEdit" name="lineEdit_ident_attribute">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Name des Identifier Attributes</string>
            </property>
            <property name="frame">
             <bool>true</bool>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Normal</enum>
            </property>
            <property name="placeholderText">
             <string>Attribut</string>
            </property>
            <property name="cursorMoveStyle">
             <enum>Qt::LogicalMoveStyle</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="button_search">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>24</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="4" colspan="2">
           <widget class="QLineEdit" name="lineEdit_ident_value">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Wert des Identifier Attributes&lt;/p&gt;&lt;p&gt;Dieser Wert darf nur einmal vorkommen.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="frame">
             <bool>true</bool>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Normal</enum>
            </property>
            <property name="placeholderText">
             <string>Wert</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_Ident">
            <property name="toolTip">
             <string>Anhand des Identifiers wird bestimmt, welche Objektvorgabe auf eine Entität angewendet werden muss</string>
            </property>
            <property name="text">
             <string>Identifier</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="ObjectTreeWidget" name="tree_object">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="dragEnabled">
           <bool>true</bool>
          </property>
          <property name="dragDropOverwriteMode">
           <bool>false</bool>
          </property>
          <property name="dragDropMode">
           <enum>QAbstractItemView::InternalMove</enum>
          </property>
          <property name="defaultDropAction">
           <enum>Qt::MoveAction</enum>
          </property>
          <property name="alternatingRowColors">
           <bool>false</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="expandsOnDoubleClick">
           <bool>false</bool>
          </property>
          <attribute name="headerShowSortIndicator" stdset="0">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Objekt</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Identifier</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Neue Spalte</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Optional</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="box_layout_pset" native="true">
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="2" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout_pSet_button">
          <item>
           <widget class="QLabel" name="label_pSet_name">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_pSet_name">
            <property name="frame">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_Pset_add">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Erstellen</string>
            </property>
            <property name="autoDefault">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QSplitter" name="splitter_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QTableWidget" name="table_pset">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="focusPolicy">
            <enum>Qt::StrongFocus</enum>
           </property>
           <property name="contextMenuPolicy">
            <enum>Qt::CustomContextMenu</enum>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
           <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
            <bool>true</bool>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderCascadingSectionResizes">
            <bool>false</bool>
           </attribute>
           <column>
            <property name="text">
             <string>PropertySet</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Geerbt von</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Optional</string>
            </property>
           </column>
          </widget>
          <widget class="QTableWidget" name="table_attribute">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <column>
            <property name="text">
             <string>Name</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Datentyp</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Format</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Wert</string>
            </property>
           </column>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1254</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Datei</string>
    </property>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>Export</string>
     </property>
     <addaction name="action_abbreviation_json"/>
     <addaction name="action_allplan"/>
     <addaction name="action_card1"/>
     <addaction name="action_vestra"/>
     <addaction name="action_excel"/>
    </widget>
    <addaction name="action_file_new"/>
    <addaction name="action_file_Open"/>
    <addaction name="action_file_Save"/>
    <addaction name="action_file_Save_As"/>
    <addaction name="action_settings"/>
    <addaction name="menuExport"/>
    <addaction name="action_mapping"/>
    <addaction name="action_project_phase"/>
    <addaction name="action_use_cases"/>
   </widget>
   <widget class="QMenu" name="menuDesite">
    <property name="toolTip">
     <string/>
    </property>
    <property name="title">
     <string>Desite</string>
    </property>
    <addaction name="action_export_bookmarks"/>
    <addaction name="action_mapping_script"/>
   </widget>
   <widget class="QMenu" name="menuPredefined_Psets">
    <property name="title">
     <string>Vordefinierte Psets</string>
    </property>
    <addaction name="action_show_list"/>
   </widget>
   <widget class="QMenu" name="menuShow_Graphs">
    <property name="title">
     <string>Bauwerksstruktur</string>
    </property>
    <addaction name="action_show_graphs"/>
   </widget>
   <widget class="QMenu" name="menu_ifc">
    <property name="title">
     <string>IFC</string>
    </property>
    <addaction name="action_modelcheck"/>
    <addaction name="action_create_groups"/>
    <addaction name="action_model_control"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDesite"/>
   <addaction name="menuPredefined_Psets"/>
   <addaction name="menuShow_Graphs"/>
   <addaction name="menu_ifc"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_file_new">
   <property name="text">
    <string>Neu</string>
   </property>
  </action>
  <action name="action_file_Save">
   <property name="text">
    <string>Speichern</string>
   </property>
  </action>
  <action name="action_file_Save_As">
   <property name="text">
    <string>Speichern unter ...</string>
   </property>
  </action>
  <action name="action_file_Open">
   <property name="text">
    <string>Öffnen</string>
   </property>
   <property name="toolTip">
    <string>Öffnet eine SOMjson Datei</string>
   </property>
  </action>
  <action name="action_desite_Settings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="action_desite_export">
   <property name="text">
    <string>Modellprüfung</string>
   </property>
   <property name="toolTip">
    <string>Erstellt eine qa.xml-Datei die in Desite als Prüfdurchlauf eingelesen werden kann. Es wird dabei die Attribuierung von Entitäten geprüft</string>
   </property>
  </action>
  <action name="action_show_list">
   <property name="text">
    <string>Anzeigen</string>
   </property>
  </action>
  <action name="action_settings">
   <property name="text">
    <string>Einstellungen</string>
   </property>
  </action>
  <action name="action_export_bs">
   <property name="text">
    <string>Export BS</string>
   </property>
  </action>
  <action name="action_export_bookmarks">
   <property name="text">
    <string>Lesezeichen</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Erstellt ein Script und eine bk.xml-Datei.&lt;/p&gt;&lt;p&gt;Es muss zuerst das Script ausgeführt  werden bevor die bk.xml-Datei importiert wird!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="action_show_graphs">
   <property name="text">
    <string>Anzeigen</string>
   </property>
   <property name="toolTip">
    <string>die Aggregationsstruktur in welcher die einzelnen Objekte gespeichert werden können</string>
   </property>
  </action>
  <action name="action_export_boq">
   <property name="text">
    <string>Export  for BoQ</string>
   </property>
  </action>
  <action name="action_mapping_options">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="action_mapping">
   <property name="text">
    <string>Mapping</string>
   </property>
   <property name="toolTip">
    <string>Revit- &amp; IFC-Mapping</string>
   </property>
  </action>
  <action name="action_shared_parameter">
   <property name="text">
    <string>Shared Parameters</string>
   </property>
  </action>
  <action name="action_ifc_mapping">
   <property name="text">
    <string>IFC Mapping</string>
   </property>
  </action>
  <action name="action_mapping_script">
   <property name="text">
    <string>Mapping Script</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Das Mappingscript verschiebt Attribute aus einem ausgewählten PropertySet in die nach SOM richtigen PropertySets&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="action_abbreviation_json">
   <property name="text">
    <string>Abbreviation JSON</string>
   </property>
  </action>
  <action name="action_allplan">
   <property name="text">
    <string>Allplan</string>
   </property>
  </action>
  <action name="action_card1">
   <property name="text">
    <string>CARD1</string>
   </property>
  </action>
  <action name="action_vestra">
   <property name="text">
    <string>Verstra</string>
   </property>
  </action>
  <action name="action_excel">
   <property name="text">
    <string>Excel</string>
   </property>
  </action>
  <action name="action_import_excel">
   <property name="text">
    <string>Excel Importieren</string>
   </property>
   <property name="toolTip">
    <string>Importiert eine SOM-MaKa Excel</string>
   </property>
  </action>
  <action name="action_modelcheck">
   <property name="text">
    <string>Modellprüfung</string>
   </property>
  </action>
  <action name="action_create_groups">
   <property name="text">
    <string>Gruppen Generieren</string>
   </property>
  </action>
  <action name="action_model_control">
   <property name="text">
    <string>Modellinformationen einlesen</string>
   </property>
  </action>
  <action name="action_project_phase">
   <property name="text">
    <string>Leistungsphase</string>
   </property>
  </action>
  <action name="actionModellpr_fung_CSV">
   <property name="text">
    <string>Modellprüfung CSV</string>
   </property>
  </action>
  <action name="action_desite_js">
   <property name="text">
    <string>Desite JavaScript</string>
   </property>
  </action>
  <action name="action_desite_csv">
   <property name="text">
    <string>Desite CSV</string>
   </property>
  </action>
  <action name="action_bim_collab">
   <property name="text">
    <string>BimCollabZoom</string>
   </property>
  </action>
  <action name="action_model_control_v2">
   <property name="text">
    <string>Modellinformationen V2</string>
   </property>
  </action>
  <action name="action_use_cases">
   <property name="text">
    <string>Anwendungsfälle</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ObjectTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>som_gui.module.objects.ui</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>line_edit_object_name</tabstop>
  <tabstop>line_edit_abbreviation</tabstop>
  <tabstop>lineEdit_ident_pSet</tabstop>
  <tabstop>lineEdit_ident_attribute</tabstop>
  <tabstop>lineEdit_ident_value</tabstop>
  <tabstop>button_objects_add</tabstop>
  <tabstop>tree_object</tabstop>
  <tabstop>button_search</tabstop>
  <tabstop>lineEdit_pSet_name</tabstop>
  <tabstop>button_Pset_add</tabstop>
  <tabstop>table_pset</tabstop>
  <tabstop>table_attribute</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
